mephisto/abstractions/providers/mturk/mturk_unit.py

```diff
@@ -162,6 +162,8 @@ def get_status(self) -> str:
         requester = self.get_requester()
         client = self._get_client(requester._requester_name)
         hit = get_hit(client, mturk_hit_id)
+        if not hit:
```

Comment:
Nit: explicitly check for `if hit is None`, as that's the only other value we expect here. Also means that after we add typechecking, not entering this condition will ensure that hit is not none, and is the other type we expect as a return value.